frontend/package.json

```diff
@@ -137,8 +137,10 @@
     "@patternfly/quickstarts": "5.1.0",
     "@patternfly/react-catalog-view-extension": "5.0.0",
     "@patternfly/react-charts": "7.1.0",
+    "@patternfly/react-console": "5.0.0",
```

Comment:
@patternfly/react-console and @patternfly/react-styles were missed and still at version 4.x under `node_modules/@patternfly/`. This improved CSS bundle sizes.